policy/pa.go

```diff
@@ -130,7 +130,7 @@ var (
 	errInvalidIdentifier   = core.MalformedRequestError("Invalid identifier type")
 	errNonPublic           = core.MalformedRequestError("Name does not end in a public suffix")
 	errICANNTLD            = core.MalformedRequestError("Name is an ICANN TLD")
-	errBlacklisted         = core.MalformedRequestError("Policy forbids issuing for name")
+	errBlacklisted         = core.RejectedIdentifierError("Policy forbids issuing for name")
```

Comment:
Please change each of the errors in this section to `probs.RejectedIdentifierProblem(...)`, and remove the definition for `core.RejectedIdentifierError`.